api/src/main/java/com/alibaba/nacos/api/config/filter/IConfigResponse.java

```diff
@@ -17,25 +17,23 @@
 
 /**
  * Config Response Interface
- * 
- * @author Nacos
  *
+ * @author Nacos
  */
 public interface IConfigResponse {
-	/**
-	 * get param
-	 * 
-	 * @param key
-	 *            key
-	 * @return value
-	 */
-	public Object getParameter(String key);
+    /**
+     * get param
+     *
+     * @param key key
+     * @return value
+     */
+    Object getParameter(String key);
 
-	/**
-	 * get context
-	 * 
-	 * @return configContext
-	 */
-	public IConfigContext getConfigContext();
+    /**
+     * get config context
```

Comment:
Capitalize the first letter

Reply:
if necessary, i will make a unified revision